apps/sms/views/conversation/test/unit/recipients_test.js

```diff
@@ -143,6 +143,24 @@ suite('Recipients', function() {
       assertDeepEqual(recipient, fixture);
     });
 
```

Comment:
I was following the format of the tests above and below it, which seem to have trailing spaces in the name. Also, where should I put the leading parenthesis?